Main.js

```diff
@@ -1368,6 +1368,13 @@ function open_player_sheet(sheet_url, closeIfOpen = true) {
 		$(event.target).contents().find(".site-bar").remove();
 		$(event.target).contents().find(".page-header").remove();
 		$(event.target).contents().find(".homebrew-comments").remove();
+		$(event.target).contents().find(".mega-menu__fallback").remove();
+    // adding styles to elements not initially available
+    window.onresize = () => {

```

Comment:
Should this be monitoring `container` or `iframe` instead of `window`?

Reply:
It should be possible to use ResizeObserver instead of onresize.

Is it desirable with [current support level](https://caniuse.com/?search=ResizeObserver) though?

Reply:
That worked, thank you. Updated the code.